modules/Components/src/client/ACASThingBrowserView.html

```diff
@@ -33,10 +33,10 @@
 <script type="text/template" id="ThingSummaryTableView" xmlns="http://www.w3.org/1999/html">
     <table class="table table-bordered table-striped table-hover" style="margin: 0px; padding: 0px; border: 0px; width: 100%; ">
         <thead style="">
-            <tr class="bv_colFilters">
-            </tr>
             <tr class = "bv_firstRow"> 
             </tr>
+            <tr class="bv_colFilters">
```

Comment:
Kindof a UX change here, to move the filter underneath the header rather than above. As long as this is intentional, I'm good with it.

Reply:
Ah whoops. Yes it was intentional. I forgot to mention it and also realize the standardization is not changed. I'm going to change the standardization to have the filters below the header as well.